qemu/tests/cfg/block_hotplug_check.cfg

```diff
@@ -17,7 +17,7 @@
     force_create_image = yes
     force_create_image_image1 = no
     soundcards = ""
-    q35:
+    q35, arm64-pci, arm64-mmio:
```

Comment:
Seems arm64-mmio does not use PCIe bridge.

Reply:
Thanks for the reminder, it has been corrected.